pkg/disk/filesystem.go

```diff
@@ -9,18 +9,19 @@ import (
 
 // Filesystem related functions
 type Filesystem struct {
-	Type string
 	// ID of the filesystem, vfat doesn't use traditional UUIDs, therefore this
-	// is just a string.
-	UUID       string
-	Label      string
-	Mountpoint string
+	Type string `json:"type"`
+
+	// UUID is just a string.
```

Comment:
The two-line comment here for UUID got broken into two.  `Type` should move one line up and the two-line comment should be above UUID.

Reply:
Thanks, nice catch! fixed.